appveyor.yml

```diff
@@ -23,20 +23,34 @@ environment:
       EXAMPLE: 01-executable
       CONFIG: Release
 
+    - TOOLCHAIN: "vs-14-2015"
+      EXAMPLE: 02-library
+      CONFIG: Release
+
+    - TOOLCHAIN: "vs-14-2015"
+      EXAMPLE: 03-shared-link
+      CONFIG: Release
+
     - TOOLCHAIN: "mingw"
       EXAMPLE: 01-executable
       CONFIG: Release
 
-    - TOOLCHAIN: "msys"
+    - TOOLCHAIN: "mingw-cxx14"
```

Comment:
I don't want to keep all toolchains in tests because there are simply too many of them. Since `mingw-cxx17` is already tested I think this one can be removed.

Reply:
Actually since `msys` seems to work I guess we can keep it.